src/ring_theory/polynomial/chebyshev/basic.lean

```diff
@@ -42,21 +43,63 @@ and do not have `map (int.cast_ring_hom R)` interfering all the time.
 noncomputable theory
 
 namespace polynomial
-open complex
 variables (R S : Type*) [comm_ring R] [comm_ring S]
 
+/-- The product of two Chebyshev polynomials is the sum of two other Chebyshev polynomials. -/
+lemma mul_chebyshev₁ :
```

Comment:
Looks good to me. I only wonder whether it makes sense to have something like
```
local notation `cheby` := chebyshev₁ R
```
so that more statements will fit within the 100-char line limit.

Reply:
Done!

Reply:
Sorry, I don't think that is a good idea. In the future, this file will most likely also contain some properties of `chebyshev\2`. If one wants to shorten it, then maybe using namespaces would be a good idea? It seems to be standard that the chebyshev polynomials are called T and U, respectively, so why not chebyshev.T and chebyshev.U and then open chebyshev in this file?

Reply:
OK, I undid the commit with the `cheby` notation.  The namespacing seems like a sensible idea, but it would be a fairly big/global change, right?  I would rather someone else took it on :)

Reply:
I would not mind doing it in a future PR if there is agreement on it.

Reply:
Ok, I agree that this sounds like a good proposal as well. It's indeed best postponed to a later PR.

Reply:
@Julian-Kuelshammer Do you want to do the file merging/re-splitting in the same follow-up PR, or do you want me to do a separate follow-up PR just for that?  Either is fine for me.

Reply:
I can do that as well if you prefer. To me it looks like the two results you worked on would go to defs while the rest of the file would go to dickson.

Reply:
OK, that sounds easy enough, thanks!  I'll do the file-moving refactor, then ping you.